release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -208,6 +208,14 @@ If you have already upgraded your cluster to {product-title} {product-version} a
 * An IPv6 network for API and Ingress services on the second `machineNetwork` configuration.
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.
+[id="ocp-4-16-networking-metallb-frr-k8s_{context}"]
+==== Integration of MetalLB and FRR-K8s (Technology Preview)
+
+This release introduces `FRR-K8s`, a Kubernetes based DaemonSet that exposes a subset of the `FRR` API in a Kubernetes-compliant manner.
+As a cluster administrator, you can use the `FRRConfiguration` custom resource (CR) to configure `MetalLB` to use `FRR-K8s` as the backend.
```

Comment:
As mentioned in the object references guidance, specific object references need to be matched with their object type, or `kind`. 
```suggestion
As a cluster administrator, you can use the `FRRConfiguration` custom resource (CR) to configure `MetalLB` <object_type> to use `FRR-K8s` <object_type> as the backend.
```

Reply:
Thanks. `MetalLB` is an operator and `FRR-K8s` is a daemon set - updated accordingly

Reply:
A friendly reminder that in the docs, "Operator" is always capitalized :)